website/src/pages/docs/getting-started/esm-typescript-usage.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: ESM TypeScript Usage
+---
+
+# ESM TypeScript Usage
+
+ESM is an important step in the JavaScript ecosystem because it allows static analysis of the depency tree.
+Forr bundlers such as Webpack this is a huge as it allows the complied bundle to only contain code that is required at runtime, which means smaller download size when loading code over the network.
+
+While ESM is a huge benefit for the whole JavaScript community, the adoption is not straight-foward.
+
+**Note:** ESM adoption in the JavaScript ecosystem is still in progress, if you choose to use ESM in your application (especially Node.js), you might hit a lot of pitfalls and incompatibility issues within libraries.
+
+If you are not interested in ESM you can easily skip this documentation.
```

Comment:
```suggestion
If you are not interested in ESM you can easily skip this documentation.
```

This is not necessary since the page's title is "ESM TypeScript Usage"

Reply:
okay I kind of assumed that people will go through each item in getting started